content/api/loaders.md

```diff
@@ -3,4 +3,302 @@ title: Loader API
 sort: 4
 ---
 
-> TODO
+Loaders allow you to preprocess files as you require() or “load” them. Loaders are kind of like “tasks” in other build tools, 
+and provide a powerful way to handle frontend build steps. Loaders can transform files from a different language like, CoffeeScript to JavaScript, or inline images as data URLs. Loaders even allow you to do things like require() css files right in your JavaScript!
+
+To tell webpack to transform a module with a loader, you can specify the loader in the webpack [configuration](../configuration) file (preferred) or in the module request, such as in a `require()` call.
```

Comment:
I don't believe those PR's have landed yet. I'll create an issue (after this merges). I'll also add a TODO. Done